pkg/agent/types/multicast.go

```diff
@@ -23,7 +23,7 @@ import (
 	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
 )
 
-type McastNPValidationItem struct {
+type McastNPRuleInfo struct {
```

Comment:
As it is rule for IGMP, we should name it IGMPNPRuleInfo?

Reply:
done